java/src/main/resources/application.yml

```diff
@@ -1,10 +1,9 @@
 spring:
   profiles:
     include:
-      - "@aws.profile.name@"
-      - "@azure.profile.name@"
-      - "@cosmosdb.profile.name@"
-      - "@dynamodb.profile.name@"
+      - aws
+      - azure
+      - dynamodb
```

Comment:
This is wrong - this needs to be an array of "@aws....@" type properties, and not literals

Reply:
fixed